package.json

```diff
@@ -118,8 +118,8 @@
                 "category": "Azure App Service"
             },
             {
-                "command": "appService.LocalGitDeploy",
-                "title": "Local Git Deploy to Web App",
+                "command": "appService.EditScmType",
```

Comment:
nit: It bothers me that the command and the title are so different

Can we merge the names to be more similar?

Reply:
I don't mind renaming it.  

The portal action is setting up a deployment source whereas the configuration object holds a scmType property.  If we want it to be more similar, I'd lean more towards Deployment Source since that is more self-explanatory than scmType.